apps/studio/data/database/view-definition-query.ts

```diff
@@ -7,10 +7,8 @@ type GetViewDefinition = {
 }
 
 export const getViewDefinitionQuery = ({ schema, name }: GetViewDefinition) => {
```

Comment:
```suggestion
export const getViewDefinitionQuery = ({ schema = 'public', name }: GetViewDefinition) => {
```
now that the `.filter(Boolean)` has been removed, we should always ensure there's a schema

Reply:
does the check at the bottom suffice though? query is enabled only if both schema and name provided

Reply:
oh true. If we're always providing a schema then that's fine